packages/node_modules/@webex/webex-core/src/lib/services/services.js

```diff
@@ -327,20 +327,49 @@ const Services = WebexPlugin.extend({
    * @returns {object}
    */
   _formatReceivedHostmap(serviceHostmap) {
-    const output = [];
-    let item = {};
+    let serviceItem;
```

Comment:
This doesn't need to be declared here. It looks like it is only used within the `map` call and the `forEach` call. You could declare it within the respective loops.